player/lua/select.lua

```diff
@@ -326,6 +298,46 @@ mp.add_key_binding(nil, "select-subtitle-line", function ()
             mp.commandv("seek", sub_times[index] + delay, "absolute")
         end,
     })
+end
+
+mp.add_key_binding(nil, "select-subtitle-line", function ()
+    local sub = mp.get_property_native("current-tracks/sub")
+
+    if sub == nil then
+        show_warning("No subtitle is loaded.")
+        return
+    end
+
+    if sub.external and sub["external-filename"]:find("^edl://") then
+        sub["external-filename"] = sub["external-filename"]:match('https?://.*')
+                                   or sub["external-filename"]
+    end
+
+    local screenx, screeny = mp.get_osd_size()
+    mp.osd_message("Subtitle export/conversion in progress.", 3)
```

Comment:
This makes sense for embedded subtitles but it is annoying to see it appear and immediately disappear for external subtitles which are extracted instantly. I would show it only `if not sub.external or sub["external-filename"]:find("://")`.

Also ↻ or ↺ may be better to indicate loading, and you could use `\an7` when `--osd-align-x` is `right` to not overlap with the text.

Reply:
> This makes sense for embedded subtitles but it is annoying to see it appear and immediately disappear for external subtitles which are extracted instantly. I would show it only `if not sub.external or sub["external-filename"]:find("://")`.

Done.

> Also ↻ or ↺ may be better to indicate loading, and you could use `\an7` when `--osd-align-x` is `right` to not overlap with the text.

Do you mean adding animated rendering? With static icons I don't think they have an advantage.

Reply:
Nah I didn't mean to animate, well whatever character you prefer is fine.